numba_cuda/numba/cuda/tests/cudadrv/test_module_callbacks.py

```diff
@@ -42,7 +41,7 @@ def get_hashable_handle_value(handle):
 
 
 @skip_on_cudasim("Module loading not implemented in the simulator")
-class TestModuleCallbacksBasic(ContextResettingTestCase):
+class TestModuleCallbacksBasic(CUDATestCase):
```

Comment:
Connected with @isVoid about these and they should be safe to remove.

Reply:
One of the reasons I left this was because I hadn't dug into why it was used, and it wasn't obvious to me - can you share the reasoning please?

Reply:
I believe it is superfluous. `context.reset` should have the effect of calling the finalizer on any modules.